src/main/java/cz/fi/muni/pa165/esports/dao/CompetitionDao.java

```diff
@@ -0,0 +1,17 @@
+package cz.fi.muni.pa165.esports.dao;
+
+import cz.fi.muni.pa165.esports.entity.Competition;
+
+import java.util.List;
+
+/**
+ * @author jan gavlik
+ */
+:
```

Comment:
Please remove this extra colon `:`, the build fails when it's there :)
```suggestion

```

Reply:
aaa thank you,